### PR TITLE
docs(zh-cn): update typo and finish translation of 'printing page'

### PR DESCRIPTION
**Pull request by jasonren0403**
### **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description
Update translation of "print page" in zh-cn.

### Motivation and Context

Provide localized text and make the selenium website more consistent

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

___

### **PR Type**
Documentation

___

### **Description**
- Completed the translation of the 'Print Page' documentation into Chinese, including the title, headings, and content.
- Provided localized descriptions for various printing configuration options such as orientation, range, size, margins, scale, background, and shrink-to-fit.
- Updated method descriptions and examples to reflect the Chinese language, ensuring consistency and accessibility for Chinese-speaking users.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>print_page.zh-cn.md</strong><dd><code>Complete Chinese translation of the 'Print Page' documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/interactions/print_page.zh-cn.md

<li>Translated the title and headings to Chinese.<br> <li> Translated the content of the document to Chinese.<br> <li> Updated method descriptions and examples to Chinese.<br> <li> Provided localized text for printing configuration options.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2052/files#diff-33cd5d0d8234827d7cacbb49b609e33192310123a67cbe0910f68a66a2b5eb6d">+28/-32</a>&nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/interactions/print_page.zh-cn.md [hunk 1]
```diff
@@ -159,8 +157,8 @@ Using `getBackground()` and `setBackground()` methods, you can get/set whether b
 {{< /tab >}}
 {{< /tabpane >}}
 
-### ShrinkToFit
-Using `getBackground()` and `setBackground()` methods, you can get/set whether the page will shrink-to-fit content on the page --- boolean `true` or `false`.
+### 缩放至合适大小
+通过 `getBackground()` 和 `setBackground()` 方法，可以获取/设置页面是否会根据页面内容缩小，其为布尔值 `true` 或 `false`。
```
Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/interactions/print_page.zh-cn.md]: This part may be also typo in English original text, should I make a commit in this PR to change that part?

```suggestion
通过 `getShrinkToFit()` 和 `setShrinkToFit()` 方法，可以获取/设置页面是否会根据页面内容缩小，其为布尔值 `true` 或 `false`。
```

**Comment by shbenzer** [comment]
@jasonren0403 Great contribution, thank you!

File: website_and_docs/content/documentation/webdriver/interactions/print_page.zh-cn.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 ]
 ---
 
-Printing a webpage is a common task, whether for sharing information or maintaining archives. 
-Selenium simplifies this process through its PrintOptions, PrintsPage, and browsingContext 
-classes, which provide a flexible and intuitive interface for automating the printing of web pages. 
-These classes enable you to configure printing preferences, such as page layout, margins, and scaling, 
-ensuring that the output meets your specific requirements.
+无论是共享信息还是维护档案，打印网页都是一项常见任务。
+Selenium 通过其 PrintOptions、PrintsPage 和 browsingContext 类简化了这一过程，这些类为自动打印网页提供了灵活直观的界面。
```
Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/interactions/print_page.zh-cn.md]:
```suggestion
Selenium 通过其 PrintOptions、PrintsPage 和 browsingContext 类简化了这一过程，这些类为网页自动打印提供了灵活直观的接口。
```

**Review by harsha509 — APPROVED** [review]
Thank you @jasonren0403 !

**Pull request merged**